README.md

```diff
@@ -1,5 +1,7 @@
 # Virality
 
+## ***WARNING***: MODS RELATING TO LARGER LOBBY SIZES CURRENTLY DO NOT WORK DUE TO SERVER LIMITATIONS
```

Comment:
`#` are meant for headings. It shouldn't be a heading.

Have a look at [Github's alert markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts).

Reply:
Yeah, I'll be adding a reference to Self Sufficient instead.